scripts/tome-sync.sh

```diff
@@ -305,10 +305,14 @@ else
   echo "No Render Dir to remove" | tee -a $TOMELOG
 fi
 
+echo "Changing directory to /tmp/ since the rend-directory we are currently in just got deleted..."
+# Note: That change-dir is done in order to stop the "aws" call below from crashing.
+cd /tmp/
+
 if [ -f "$TOMELOG" ]; then
   echo "Saving logs of this run to S3: $TOMELOG -> $BUCKET_NAME/tome-log/$TOMELOGFILE" | tee -a $TOMELOG
-  echo "SYNC FINISHED" | tee -a $TOMELOG
   aws s3 cp $TOMELOG s3://$BUCKET_NAME/tome-log/$TOMELOGFILE $S3_EXTRA_PARAMS 2>&1 | tee -a $TOMELOG
+  echo "SYNC FINISHED" | tee -a $TOMELOG
```

Comment:
I have to admit I haven't tried to test this locally but ... why move this "SYNC FINISHED" statement? We want to write to the log before we copy the log to s3, I think. Was it breaking somehow?

Reply:
Ah, after looking at this more, yeah, it makes sense to leave it as-is. I thought the "Finished" message shouldnt show until the S3 sync was done - but yeah, that finish-message needs to be in the log.
I have restored that line back to what it was now.